pytest-selenium/pages/content.py

```diff
@@ -127,11 +127,50 @@ def click_previous_link(self):
     class NavBar(Region):
         _root_locator = (By.CSS_SELECTOR, '[data-testid="navbar"]')
         _openstax_logo_link_locator = (By.CSS_SELECTOR, "div > a")
+        _user_nav_locator = (By.CSS_SELECTOR, '[data-testid="user-nav"]')
+        _login_locator = (By.CSS_SELECTOR, "[data-testid='nav-login']")
+        _user_nav_toggle_locator = (By.CSS_SELECTOR, '[data-testid="user-nav-toggle"]')
+        _account_profile_locator = (By.XPATH, "//a[contains(text(), 'Account Profile')]")
+        _logout_locator = (By.XPATH, "//a[contains(text(), 'Log out')]")
```

Comment:
Is there a style preference within the tests for single- or double-quoted strings?

Reply:
Updated the current test to have double quotes for the locators. will maintain uniformity for future tests as well